tools/pkg/commands/install.toit

```diff
@@ -0,0 +1,177 @@
+
+  execute-remote:
+    remote-package := registries.search package
+    if not prefix: prefix = remote-package.name
+
+    if project.package-file.has-package prefix:
+      error "Project already has a package with name '$prefix'."
+
+    project.install-remote prefix remote-package
+
+  execute-local:
+    package-file-name := "$package/$PackageFile.FILE_NAME"
+    src-directory := "$package/src"
+    if not file.is-file package-file-name:
+      error "Path supplied in package argument is an invalid local package, missing $package-file-name."
+
+    if not file.is-directory src-directory:
+      error "Path supplied in package argument is an invalid local package, missing $src-directory."
+
+    package-file := ExternalPackageFile (fs.to-absolute package)
+    if not prefix: prefix = package-file.name
+
+    project.install-local prefix package
+
+  static CLI-COMMAND ::=
+      cli.Command "install"
+          --help="""
+              Installs a package in the current project, or downloads all dependencies.
+
+              If no 'package' is given, then the command downloads all dependencies.
+                If necessary, updates the lock-file. This can happen if the lock file doesn't exist
+                yet, or if the lock-file has local path dependencies (which could have their own
+                dependencies changed). Recomputation of the dependencies can also be forced by
+                providing the '--recompute' flag.
+
+              If a 'package' is given finds the package with the given name or URL and installs it.
+                The given 'package' string must uniquely identify a package in the registry.
+                It is matched against all package names, and URLs. For the names, a package is considered
+                a match if the string is equal. For URLs it is a match if the string is a complete match, or
+                the '/' + string is a suffix of the URL.
+
+              The 'package' may be suffixed by a version with a '@' separating the package name and
+                the version. The version doesn't need to be complete. For example 'foo@2' installs
+                the package foo with the highest version satisfying '2.0.0 <= version < 3.0.0'.
+                Note: the version constraint in the package.yaml is set to accept semver compatible
+                versions. If necessary, modify the constraint in that file.
+
+              Installed packages can be imported by their prefix. By default the prefix is their
+                name, but the '--prefix' argument can override the default.
+
+              If the '--local' flag is used, then the 'package' argument is interpreted as
+                a local path to a package directory. Note that published packages may not
+                contain local packages.
+              """
+          --examples=[
+              cli.Example --arguments=""
+                  """
+                  Assumes that the package 'toitware/toit-morse' has the
```

Comment:
This help-text doesn't seem to match the example.

Looked at the original code. The "Assumes" was intended for all examples. That's probably a bad idea anyway.